### PR TITLE
Implemented two_sum in functions and tests

### PR DESCRIPTION
**Pull request by kira-kaur-sidhu**
Please review!

### CODE REVIEW

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
```
Comment (qicarolyn1) [inline functions/two_sum.py:1]: Can you add some documentation here?

File: tests/two_sum_test.py [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+def test_returns_0_1_for_3_3_with_target_6():
+    # Arrange
+    n = [3, 3]
+    t = 6
+
+    # Act
+    answer = twoSum(n, t)
+
+    # Assert
+    assert answer == [0, 1]
```
Comment (qicarolyn1) [inline tests/two_sum_test.py:37]: You should include test cases with edge cases

**Review by qicarolyn1 — APPROVED** [review]
woot woot looks good.
